python/pyspark/sql/datasource.py

```diff
@@ -539,6 +539,11 @@ def pushFilters(self, filters: List["Filter"]) -> Iterable["Filter"]:
         This method is allowed to modify `self`. The object must remain picklable.
         Modifications to `self` are visible to the `partitions()` and `read()` methods.
 
+        Notes
+        -----
+        Configuration `spark.sql.python.filterPushdown.enabled` must be set to `true`
+        to implement this method.
```

Comment:
Not sure if we should put this in the doc. Can we throw an warning in the code?

Reply:
Yeah, we already show an error if it's disabled. I guess the user can find out when they try to use the source so it's not necessary to put in the doc.

Reply:
let's remove it from the doc